appveyor.yml

```diff
@@ -39,7 +39,7 @@ install:
   - "conda create -q -n test-environment python=%PYTHON% numpy scipy matplotlib cython jupyter ipython pillow wheel"
   - activate test-environment
   - pip install -r requirements_dev.txt
-  - python setup.py install
+  - pip install git+git://github.com/simpeg/geoana.git@magnetic_pole
```

Comment:
@lheagy I am not sure why ..., but in appveyor it seem we are using geoana in pypi ... 
this passes the test